From f22cf49308b16a26e4d41eeac34eaecc79490234 Mon Sep 17 00:00:00 2001
From: Ranjani Vaidyanathan <ra5478@freescale.com>
Date: Wed, 1 Aug 2012 17:42:29 -0500
Subject: [PATCH 1855/2508] ENGR000219304 Read the correct chip version for
 mx6DL and MX6SL

Use the correct function to read the chip revision.

Signed-off-by: Ranjani Vaidyanathan <ra5478@freescale.com>
---
 arch/arm/mach-mx6/cpu.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/mach-mx6/cpu.c b/arch/arm/mach-mx6/cpu.c
index 9a60d98..be5011a 100644
--- a/arch/arm/mach-mx6/cpu.c
+++ b/arch/arm/mach-mx6/cpu.c
@@ -201,9 +201,9 @@ static int __init post_cpu_init(void)
 	if (cpu_is_mx6q())
 		chip_rev = mx6q_revision();
 	else if (cpu_is_mx6dl())
-		chip_rev = mx6dl_revision;
+		chip_rev = mx6dl_revision();
 	else
-		chip_rev = mx6sl_revision;
+		chip_rev = mx6sl_revision();
 
 	return 0;
 }
-- 
1.7.7.4

